src/xcov-core/Core/Writer.h

```diff
@@ -9,6 +9,6 @@
 @interface Writer : NSObject
 
 - (instancetype)initWithOptions:(CoreOptions)options;
-- (void)writeReport:(NSDictionary*)report;
+- (void)writeReport:(NSDictionary *)report;
```

Comment:
Ok, here I don't know... 😃
There are many places where there is a space between parameter type and `*`.
For example [this one](https://github.com/nakiostudio/xcov-core/blob/master/src/xcov-core/Middleware.m#L63).

Maybe I pay too much attention to consistency, but I think we can decide on some sort of coding style in order to avoid future ping-pongs of this kind. 

Until we decide on this, I'll reverd these changes as well. `¯\_(ツ)_/¯`